src/sqlfluff/core/templaters/python.py

```diff
@@ -965,20 +976,58 @@ def _split_uniques_coalesce_rest(
                         # Anything up to start_idx (exclusive) is included.
                         include_start = raw_idx > elem_sub_buffer[0][2]
 
-                        # The ending point of this slice, is already decided.
-                        end_point = elem_sub_buffer[-1].end_source_idx()
-
```

Comment:
Didn't really remove this - just moved this down a bit, after we finish dealing with the start.

Reply:
Reverted so resolving.